templates/views/header.html

```diff
@@ -8,7 +8,12 @@
             </i>
         </div>
         <div class="f-brand item fitted">
-            <img class="f-logo" alt="Forsta" src="{{static 'images/logo_just_text.svg'}}"/>
+            {{#if orgAttrs.banner}}
```

Comment:
see prior comments, you just put templating stuff inside the tag itself, handlebars doesn't know html from shakespeare, so it doesn't care where you put the expressions.